Makefile

```
@@ -1,13 +1,20 @@
 VERSION = 2
 LIBRARY_NAME = pam_watchid.so
 DESTINATION = /usr/local/lib/pam
-TARGET = x86_64-apple-macosx10.15
+ARCH := $(shell uname -m)
+ifeq ($(ARCH), arm64)
+TARGET := arm64-apple-darwin20.1.0
+else
+TARGET := x86_64-apple-darwin20.1.0
+endif
```

Why don't use:

```suggestion
TARGET := $(shell uname -m)-apple-darwin$(shell uname -r)
```

This will be architecture and platform independent.

---

Thanks for the suggestion, my change comes from the upstream repo.